connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -492,7 +493,14 @@ public void putSessionKey(SessionKey sessionKey) {
             public void run() {
                 log.debug("Creating admin client to manage Connect internal config topic");
                 try (TopicAdmin admin = new TopicAdmin(adminProps)) {
-                    admin.createTopics(topicDescription);
+                    // Create the topic if it doesn't exist
+                    Set<String> newTopics = admin.createTopics(topicDescription);
```

Comment:
Vararg gives some unintended consequences in naming. Should we stick to singular given that we expect at most a single topic to be created?